src/manage/delete-files/index.md

```diff
@@ -2,25 +2,27 @@
 
 Deleting files such as photos that are not needed anymore can help you free up storage on the <MainPlatformNameLink /> cloud and keep your project organised.
 
-Files can be deleted through <AppDomainNameLink /> one by one. Deleting multiple files at once is possible on your computer using <QGISPluginName />.
+Files can be deleted through <DashboardLink /> one by one. Deleting multiple files at once is possible on your computer using <QGISPluginName />.
 [[toc]]
 
 :::warning
 Download and archive the data before deleting them if they may be needed in the future.
 :::
 
-## Delete files through merginmaps.com
-Deleting files trough <MainDomainName /> is simple. However, you can only delete one file at the time.
+## Delete files through Mergin Maps dashboard
+Deleting files through <MainDomainName /> is simple. However, you can only delete one file at the time.
```

Comment:
Edit text:
Deleting files through `<MainDomainName />` is simple. However, you can only delete one file at a time.